README.md

```diff
@@ -13,34 +13,67 @@ The Open Neural Network Exchange implementation in MLIR (http://onnx.ai/onnx-mli
-launch a container and run an interactive bash shell in it, or use it as the base image in a dockerfile. The container contains the full build tree including
-the prerequisites and a clone of the source code. The source can be modified and onnx-mlir rebuilt from within the container, so it is possible to use it
-as a development environment. It is also possible to attach vscode to the running container. An example Dockerfile and vscode configuration files can be
-seen in the docs folder. The Dockerfile is shown here.
+Currently there are both Release and Debug mode images for amd64, ppc64le and IBM System Z saved in Docker Hub as, respectively, onnxmlirczar/onnx-mlir
+and onnxmlirczar/onnx-mlir-dev. To use one of these images either pull it directly from Docker Hub,
+launch a container and run an interactive bash shell in it, or use it as the base image in a dockerfile. The onnx-mlir container just contains the 
+built compiler and can be used to compile models. The onnx-mlir-dev container contains the full build tree including the prerequisites and a clone
+of the source code. The source can be modified and onnx-mlir rebuilt from within the container, so it is possible to use it
+as a development environment. It is also possible to attach vscode to the running container. An example Dockerfile useful for development and vscode 
+configuration files can be seen in the docs folder. If the workspace directory and the vscode files are not present in the directory where the Docker build is run, then the lines 
+referencing them should be commented out or deleted. The Dockerfile is shown here.
 
 [same-as-file]: <> (docs/docker-example/Dockerfile)
 ```
-FROM onnxmlirczar/onnx-mlir-build:amd64
+FROM onnxmlirczar/onnx-mlir-dev
 
-WORKDIR /build
-ENV HOME=/build
-ENV PYENV_ROOT=$HOME/.pyenv
-ENV PATH=$PYENV_ROOT/shims:$PYENV_ROOT/bin:$PATH
-RUN pyenv global 3.7.0
-RUN pyenv rehash
+WORKDIR /workdir
+ENV HOME=/workdir
 
-ENV PATH=$PATH:/build/bin
+# install needed packages
+ENV PATH=$PATH:/workdir/bin
 RUN apt-get update
 RUN apt-get install -y python-numpy
 RUN apt-get install -y python3-pip
+RUN python -m pip install --upgrade pip
 RUN apt-get install -y gdb
 RUN apt-get install -y lldb
 RUN apt-get install -y emacs
-WORKDIR /build/.vscode
-ADD .vscode /build/.vscode
-WORKDIR /build
+RUN apt-get install -y libeigen3-dev
+RUN apt-get install -y clang-format
+# install clang-12
+RUN apt-get install -y lsb-release wget software-properties-common
+RUN bash -c "$(wget -O - https://apt.llvm.org/llvm.sh)"
+RUN python -m pip install wheel
+RUN python -m pip install numpy
+RUN python -m pip install torch==1.6.0+cpu torchvision==0.7.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
+RUN git clone https://github.com/onnx/tutorials.git
+
+
+# set env variables for make - these should be set in any shell that is started
+ENV LLVM_PROJ_SRC=/workdir/llvm-project
```

Comment:
I don't think LLVM_PROJ_SRC and LLVM_PROJ_BUILD are needed any more, are they? The CMake now requires a path to MLIR_DIR instead.

Reply:
Addressed this with PR #710